fe/fe-core/src/test/resources/sql/scheduler/tpch/q9.sql

```diff
@@ -0,0 +1,530 @@
+        0:OlapScanNode
+          1. partitionID=1362,tabletID=1367
+          2. partitionID=1362,tabletID=1373
+          3. partitionID=1362,tabletID=1379
+          4. partitionID=1362,tabletID=1385
+          5. partitionID=1362,tabletID=1391
+          6. partitionID=1362,tabletID=1397
+    INSTANCE(24-F00#1)
+      DESTINATIONS: 14-F06#0,15-F06#1,16-F06#2
+      BE: 10002
+      SCAN RANGES
+        BUCKET SEQUENCES: [17, 2, 5, 8, 11, 14]
+        0:OlapScanNode
+          1. partitionID=1362,tabletID=1369
+          2. partitionID=1362,tabletID=1375
+          3. partitionID=1362,tabletID=1381
+          4. partitionID=1362,tabletID=1387
+          5. partitionID=1362,tabletID=1393
+          6. partitionID=1362,tabletID=1399
+    INSTANCE(25-F00#2)
+      DESTINATIONS: 14-F06#0,15-F06#1,16-F06#2
+      BE: 10003
+      SCAN RANGES
+        BUCKET SEQUENCES: [0, 3, 6, 9, 12, 15]
+        0:OlapScanNode
+          1. partitionID=1362,tabletID=1365
+          2. partitionID=1362,tabletID=1371
+          3. partitionID=1362,tabletID=1377
+          4. partitionID=1362,tabletID=1383
+          5. partitionID=1362,tabletID=1389
+          6. partitionID=1362,tabletID=1395
+
+PLAN FRAGMENT 10(F01)
+  DOP: 16
+  INSTANCES
+    INSTANCE(26-F01#0)
+      DESTINATIONS: 25-F00#2,23-F00#0,24-F00#1,25-F00#2,23-F00#0,24-F00#1,25-F00#2,23-F00#0,24-F00#1,25-F00#2,23-F00#0,24-F00#1,25-F00#2,23-F00#0,24-F00#1,25-F00#2,23-F00#0,24-F00#1
+      BE: 10001
+      SCAN RANGES
+        1:OlapScanNode
+          1. partitionID=1001,tabletID=1004
+          2. partitionID=1001,tabletID=1010
+          3. partitionID=1001,tabletID=1016
+          4. partitionID=1001,tabletID=1022
+          5. partitionID=1001,tabletID=1028
+          6. partitionID=1001,tabletID=1034
+          7. partitionID=1001,tabletID=1040
+    INSTANCE(27-F01#1)
+      DESTINATIONS: 25-F00#2,23-F00#0,24-F00#1,25-F00#2,23-F00#0,24-F00#1,25-F00#2,23-F00#0,24-F00#1,25-F00#2,23-F00#0,24-F00#1,25-F00#2,23-F00#0,24-F00#1,25-F00#2,23-F00#0,24-F00#1
+      BE: 10002
+      SCAN RANGES
+        1:OlapScanNode
+          1. partitionID=1001,tabletID=1006
+          2. partitionID=1001,tabletID=1012
+          3. partitionID=1001,tabletID=1018
+          4. partitionID=1001,tabletID=1024
+          5. partitionID=1001,tabletID=1030
+          6. partitionID=1001,tabletID=1036
+          7. partitionID=1001,tabletID=1042
+    INSTANCE(28-F01#2)
+      DESTINATIONS: 25-F00#2,23-F00#0,24-F00#1,25-F00#2,23-F00#0,24-F00#1,25-F00#2,23-F00#0,24-F00#1,25-F00#2,23-F00#0,24-F00#1,25-F00#2,23-F00#0,24-F00#1,25-F00#2,23-F00#0,24-F00#1
+      BE: 10003
+      SCAN RANGES
+        1:OlapScanNode
+          1. partitionID=1001,tabletID=1008
+          2. partitionID=1001,tabletID=1014
+          3. partitionID=1001,tabletID=1020
+          4. partitionID=1001,tabletID=1026
+          5. partitionID=1001,tabletID=1032
+          6. partitionID=1001,tabletID=1038
+
+[fragment]
```

Comment:
In the scheduler test, I think we don't need to output the plan result.

Reply:
The command `explain scheduler` only shows the distribution of fragment instance, but doesn't show the plan. 
Therefore, if the test case doesn't pass, maybe we could use the plan and scheduler information to troubleshoot.